### ISSUE TITLE
[bug][data] jba3.0 ebrains info missing

### ISSUE THREAD

**Issue by xgui3783** [FZJ-INM1-BDA/siibra-python#387]
as of 5a649faf94f07abcd91c0c8979a58bcd07b9c7bf

example code:

```python
import siibra
m = siibra.get_map("3.0", "mni152", "statistical")
assert any([len(v.datasets) > 0 for v in m.volumes])
```

expected: some volume have datasets (ebrains links)

actual: none do

related to: https://github.com/FZJ-INM1-BDA/siibra-explorer/issues/1322

**Comment by AhmetNSimsek** [comment]
Is there a table for which we can extract this information for jba3 maps?

**Comment by xgui3783** [comment]
previously we manually curate this list. see https://jugit.fz-juelich.de/t.dickscheid/brainscapes-configurations/-/blob/siibra-0.3a27/parcellations/julichbrain_v2_9_0.json#L5951-5955

I ported them to jba2.9 in statistical map. see https://jugit.fz-juelich.de/t.dickscheid/brainscapes-configurations/-/blob/siibra-0.4a46/maps/colin27-jba29-continuous.json#L935-938

I suspect similar arrangements will need to be made for jba3.0

**Comment by AhmetNSimsek** [comment]
There may be a work around using the kg query. I talked to Lyuba about this and will try to automate it.

**Comment by AhmetNSimsek** [comment]
https://jugit.fz-juelich.de/t.dickscheid/brainscapes-configurations/-/commit/45695b3b318138296fd6e2ffb187eeeda99ca10e is expected to solve this issue. Manual testing is required.

**Comment by xgui3783** [comment]
fixed as of v0.4a61
